### PR TITLE
Add raw output to the OCR scanner

### PR DESCRIPTION
**Pull request by rw-access**
**Describe the change**
Add raw output to the OCR scanner, so we preserve whitespace.
Preserve the split `text` field.

Also fixed the dockerfiles because they were failing due to Linux distro changes.

**Describe testing procedures**
Tested with docker locally, inspected output to confirm the field showed as expected.

**Sample output**
Scanned screenshot of the python tesseract source code this in the sublime platform. Whitespace is there, but you don't see it because our frontend strips it all out.

Screen Shot 2022-08-19 at 5 26 07 PM

**Checklist**
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of and tested my code
- [ ] ~I have commented my code, particularly in hard-to-understand areas~
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: build/python/backend/Dockerfile [hunk 1]
```diff
@@ -94,13 +88,21 @@ RUN apt-get -qq update && \
...
+    libpcap-dev \
+    libbz2-dev \
+    libgomp1
 # The maintainer isn't big on releases or tags so grab an arbitrary, but consistent, commit.
 # Additionally jump through some extra hoops to get the single commit to save some download time.
 RUN mkdir jtr && cd jtr && git init && git remote add origin https://github.com/openwall/john.git && git fetch --depth 1 origin b5c10480f56ff1b5d76c6cbdaf9c817582ee2228 && git reset --hard FETCH_HEAD \
   && rm -rf /jtr/.git \
   && cd /jtr/src \
-  && ./configure --disable-native-tests \
+  && ./configure \
```
Comment (cameron-dunn-sublime) [inline build/python/backend/Dockerfile:105]: Is there a reason to get rid of this?

https://github.com/sublime-security/strelka/pull/21

I don't think we should unless we've confirmed we won't run into the issues this fixed.

File: build/python/backend/Dockerfile [hunk 2]
```diff
@@ -148,4 +150,5 @@ COPY ./build/configs/taste.yara /strelka/config
 COPY ./build/configs/passwords.dat /strelka/config
 COPY ./build/configs/scanners.yaml /strelka/config
 
-USER 1001
+
+USER 1001
```
Comment (cameron-dunn-sublime) [inline build/python/backend/Dockerfile:154]: Nit, add a new line after this to make git happy

File: build/python/mmrpc/Dockerfile [hunk 3]
```diff
@@ -38,4 +38,4 @@ RUN cd /strelka/ && \
     apt-get clean -qq && \
     rm -rf /var/lib/apt/lists/* /strelka/
 
-USER 1001
+USER 1001
```
Comment (cameron-dunn-sublime) [inline build/python/mmrpc/Dockerfile:41]: Same nit

File: build/python/backend/Dockerfile [hunk 4]
```diff
@@ -1,19 +1,11 @@
...
-ARG CAPA_VERSION=3.0.3
-ARG EXIFTOOL_VERSION=12.38
-
-RUN apt-get update -y && apt-get install -y \
-    # expat: CVE-2022-23852 & CVE-2022-23990;
-    expat=2.4.1-2ubuntu0.3 \
-    # mount & util-linux: CVE-2021-3996 & CVE-2021-3995
-    mount=2.36.1-8ubuntu2.2 util-linux=2.36.1-8ubuntu2.2 \
-    # libsepol: CVE-2021-36087 & USN-5391-1 & CVE-2021-36084 & CVE-2021-36086
-    libsepol1=3.1-1ubuntu2.1 sepol-utils=3.1-1ubuntu2.1
```
Comment (cameron-dunn-sublime) [inline build/python/backend/Dockerfile:16]: I'm assuming that some or all of these were incompatible with `ubuntu:22.10`? These were all added to address specific issues we had flagged, but we can just see if they still require mitigation.

Comment (rw-access) [reply build/python/backend/Dockerfile:16]: I kinda blindly copied this file. I can restore this and the other deleted lines

**Review by cameron-dunn-sublime — APPROVED** [review]

**Pull request merged**